### PR TITLE
Add ships

### PR DESCRIPTION
**Pull request by Rahul2325**
Implemented addShips Func

### CODE REVIEW

**Review by ChandaSukesh — COMMENTED** [review]
Good

**Comment by ChandaSukesh** [comment]
Good. Try to maintain a proper naming conversions.

File: battleship.py [hunk 1]
```diff
@@ -114,7 +114,15 @@ def checkShip(grid, ship):
 Returns: 2D list of ints
 '''
 def addShips(grid, numShips):
-    return
+    board=grid
```
Comment (ChandaSukesh) [inline battleship.py]: This assignment is irrelevant and incorrect. Please work on the grid variable directly.

**Review by LaxmiNarayanaMurthyVemuri — APPROVED** [review]
Good job @Rahul2325

**Comment by Rahul2325** [comment]
I have merged all the 4 functions which are in the addShips branch to the main branch.

**Pull request merged**